liquid-integration/src/main/java/com/e/liquid_integration/adapter/CryptoCurrencyAdapter.kt

```diff
@@ -0,0 +1,68 @@
+package com.e.liquid_integration.adapter
```

Comment:
package should be `org.dash.wallet.integration.liquid.*` if we follow the same convention as with Uphold.